protos/calibration/calibration.proto

```diff
@@ -10,6 +10,7 @@ service CalibrationService {
     rpc SubscribeCalibrateAccelerometer(SubscribeCalibrateAccelerometerRequest) returns(stream CalibrateAccelerometerResponse) {}
     rpc SubscribeCalibrateMagnetometer(SubscribeCalibrateMagnetometerRequest) returns(stream CalibrateMagnetometerResponse) {}
     rpc SubscribeCalibrateGimbalAccelerometer(SubscribeCalibrateGimbalAccelerometerRequest) returns(stream CalibrateGimbalAccelerometerResponse) {}
+    rpc Cancel(CancelCalibrationRequest) returns(CancelCalibrationResponse) {}
```

Comment:
Does that work :thinking: ? I believe that my templates assume that it's the same prefix everywhere:

`Cancel`, `CancelRequest`, `CancelResponse`, or
`CancelCalibration`, `CancelCalibrationRequest`, `CancelCalibrationResponse`

Then I think I would vote for `Cancel`, because it will then become `calibration.cancel()` instead of `calibration.cancelCalibration()`.

Reply:
Got it. I did have to modify something in the generated SDK. I'll see what the consensus is in the other channel about the naming convention and change it here. :)